### DISCUSSION TITLE
Desabilitar a função que faz abrir o browser ao salvar

### DISCUSSION THREAD

**Discussion by eliumoraes** [balta-io/7194#21] [Perguntas e Respostas] [answered]
Gostaria de deixar marcado, caso alguém passe por isso, que é possível desabilitar o lançamento do navegador acessando o arquivo .vscode\launch.json e comentando as seguintes linhas:

`
"serverReadyAction": {
    "action": "openExternally",
    "pattern": "\\bNow listening on:\\s+(https?://\\S+)"
},
`

**Comment by andrebaltieri** [comment, marked as answer]
Muito obrigado @eliumoraes
